mRemoteV1/Connection/ConnectionInfoInheritance.cs

```diff
@@ -83,9 +83,14 @@ public bool EverythingInherited
 		[LocalizedAttributes.LocalizedCategory("strCategoryProtocol", 4), 
 		LocalizedAttributes.LocalizedDisplayNameInheritAttribute("strPropertyNameAuthenticationLevel"), 
 		LocalizedAttributes.LocalizedDescriptionInheritAttribute("strPropertyDescriptionAuthenticationLevel"), 
-		TypeConverter(typeof(MiscTools.YesNoTypeConverter))]public bool RDPAuthenticationLevel {get; set;}
-				
-		[LocalizedAttributes.LocalizedCategory("strCategoryProtocol", 4), 
+		TypeConverter(typeof(MiscTools.YesNoTypeConverter))]public bool RDPAuthenticationLevel {get; set; }
+
+        [LocalizedAttributes.LocalizedCategory("strCategoryProtocol", 4),
+        LocalizedAttributes.LocalizedDisplayNameInheritAttribute("strPropertyNameRDPMinutesToIdleTimeout"),
+        LocalizedAttributes.LocalizedDescriptionInheritAttribute("strPropertyDescriptionRDPMinutesToIdleTimeout"),
```

Comment:
Here's the source of the issue for my last requested change (name/description not showing up in the config window).

You have `strPropertyNameRDPMinutesToIdleTimeout` set for the attribute on line 89, though the actual name for the localized string is `strPropertyNameMinutesToIdleTimeout`. Same for line 90. This will need to be changed to pull in the correct (non-empty) name and description.